### DISCUSSION TITLE
Syntax Error: no identifiers allowed directly after numeric literal

### DISCUSSION THREAD

**Discussion by alicesin615** [wevm/wagmi#3234] [Q&A]
I'm using wagmi ver 1.4.7  for my react 17 web application and webpack version 5 as the bundler. Was doing some cross-browser testing and realised that in older browsers such as ios safari 13, i run into this javascript exception syntax error "no identifiers allowed directly after numeric literal" and based on my initial investigation, the source seems to be coming from wagmi.

**Comment by alicesin615** [comment]
Was doing further research on this issue, seems that a typical approach of using babel and its plugins to handle the transpilation is not going to work here because there's currently no plugin that can support the above mentioned syntax error.

**Comment by Faisalnawaz953** [comment]
@alicesin615   any solution?i am still getting error in react-native when using wallet connect v3 verison with wagmi if i comment code  related to wagmi it works fine else it gives error


````
import {polygonMumbai, bscTestnet, goerli} from 'viem/chains';
import {
  createWeb3Modal,
  defaultWagmiConfig,
  Web3Modal,
} from '@web3modal/wagmi-react-native';
const chains = [polygonMumbai, goerli, bscTestnet];
const wagmiConfig = defaultWagmiConfig({
    chains: [],
    projectId: projectId,
    metadata: providerMetadata,
  });
createWeb3Modal({
  projectId: projectId,
  chains: [],
  wagmiConfig: wagmiConfig,
});

const App=()=>{

return (
    <GestureHandlerRootView style={{flex: 1}}>
      <WagmiConfig config={wagmiConfig}> 
         <Web3Modal /> 

        <Provider store={store}>
          <PersistGate
            loading={<AppText>Loading</AppText>}
            persistor={persistor}>
            <SafeAreaProvider style={styles.container}>
              <StatusBar translucent backgroundColor={COLORS.TRANSPARENT} />
              <RootNavigation />
            </SafeAreaProvider>
          </PersistGate>
        </Provider>
        <Toast config={toastConfig} />
       </WagmiConfig> 
    </GestureHandlerRootView>
  );
}
